README.md

```diff
@@ -26,7 +26,94 @@ Estimating the geographical range of a species from sparse observations is a cha
 ```
 
 #### Data Download and Preparation
```

Comment:
Please add titles for the two sections so there's "set up instructions" and "run the code" (or something like that), since we are making the readme file longer let's make it easier to navigate.

Reply:
new changes